services/address_book_service.proto

```diff
@@ -29,37 +27,132 @@ import "transaction_response.proto";
 import "transaction.proto";
 
 /**
- * Transactions for the AddressBook Service, those HAPI APIs facilitate changes to the nodes used across the Hedera network.
- * All those transactions needs to be signed by the Hedera Council. Steps needed below:
- * 1. The node operator creates and signs the transaction with their key (the key on the node operator account)
- * 2. The node operator hands this transaction to Alex, who then gives it to the council to sign
- * 3. When signed and submitted, the server will verify that account 2 keys have signed, and the keys on the operator account have signed.
- * Hedera council should have ability to make all edits in addition to add/remove nodes
+ * The Address Book service provides the ability for Hedera network node
+ * administrators to add, update, and remove consensus nodes. This addition,
+ * update, or removal of a consensus node requires governing council approval,
+ * but each node operator may update their own operational attributes without
+ * additional approval, reducing overhead for routine operations.
+ *
+ * Most operations are `privileged operations` and require governing council
+ * approval.
+ *
+ * ### For a node creation transaction.
+ * - The node operator SHALL create a `createNode` transaction.
+ *    - The node operator SHALL sign this transaction with the active `key` for
+ *      the account to be assigned as the "node account".
+ *    - The node operator MUST deliver the signed transaction to the Hedera
+ *      council representative.
+ *    - The Hedera council representative SHALL arrange for council members to
+ *      review and sign the transaction.
+ *    - Once sufficient council members have signed the transaction, the
+ *      Hedera council representative SHALL submit the transaction to the
+ *      network.
+ * - Upon receipt of a valid and signed node creation transaction the network
+ *   software SHALL
+ *    - Validate the threshold signature for the Hedera governing council
+ *    - Validate the signature of the active `key` for the account to be
+ *      assigned as the "node account".
+ *    - Create the new node in a pending state.
```

Comment:
Maybe remove '"pending state' here, same below 'transition the new pending node to an active status'? We can say "the new node is not active until next freeze updrade' or something like that.

Reply:
Updated here, and also for the similar items in update and delete.